tests/htmlMethods.spec.js

```diff
@@ -14,10 +14,23 @@ describe('Entity without declaration', () => {
         expect(shallow(<Bem block="Block" tag="b"/>).type()).toBe('b');
     });
 
-    it('Should proper attrs', () => {
-        expect(shallow(<Bem block="Block" tag="b" attrs={{ id : 'the-id' }}/>).prop('id'))
+    it('Should proper inline attrs', () => {
+        expect(shallow(<Bem block="Block" tag="b" id="the-id"/>).prop('id'))
             .toBe('the-id');
     });
+
+    it('Should merge inline attrs and attrs mode', () => {
+        expect(shallow(<Bem block="Block" id="the-id" attrs={{ action : '/' }}/>).props())
+            .toMatchObject({
+                id : 'the-id',
+                action : '/'
+            });
+    });
+
+    it.skip('Should throw error in attrs conflict', () => {
+        expect(shallow(<Bem block="Block" id="the-id" attrs={{ id : 'no-id' }}/>))
```

Comment:
So, I really don't know what we need to do in this case ;(

Reply:
We should decide what is more important. Feels like `id` should be more specific than `attrs.id`.

Reply:
Maybe yes ;) But I don't understand why we need to support `attrs` here. Because you can add attrs like this:
``` js
<Bem {{block: "Block", id: "the-id", ...props.attrs}}/>
```